perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -75,10 +75,16 @@ def _Create(self):
         image_id = image.id
         boot_from_vol = []
         if FLAGS.openstack_boot_from_volume:
+
+            if FLAGS.openstack_volume_size > 0:
+                volume_size = FLAGS.openstack_volume_size
+            else:
+                volume_size = flavor.disk
+
```

Comment:
I think logging is required to add to whole OpenStack driver then we can make separated story for that.